index.js

```diff
@@ -478,23 +483,46 @@ function modifyFirstrun() {
     });
 }
 
+/**
+ * Overrides time interval defauls from config file.
+ */
+function overrideDefaults() {
+    try {
+        // load the config file
+        let config = self.data.load('./config.json');
```

Comment:
This doesn't seem to be working for me. The file location I've got is /data/config.json. When I move the file to the parent directory (the root of the entire project) and change this line to ` let config = self.data.load('./../config.json');` it seems to be working. Not sure why.